microsoft-edge/devtools-guide-chromium/whats-new/2021/07/devtools.md

```diff
@@ -0,0 +1,288 @@
+   DevTools extension integrated with Visual Studio Code Debugger workflow
+:::image-end:::
+
+For more information, navigate to [Launching Edge DevTools from the JS Debugger workflow][GithubVscodeEdgeDevtoolsDebuggerIntegration].
+<!-- todo: link to PR articles if available (soon).  Else the github page will link to those -->
+<!-- no crbug -->
+
+<!-- standard content for extension items: -->
+Get the [Microsoft Edge Developer Tools for Visual Studio Code extension][VisualstudioMarketplaceMsEdgedevtoolsVscodeEdgeDevtools].  Microsoft Visual Studio Code updates extensions automatically.  To manually update the version, navigate to [Update an extension manually][VisualstudioCodeDocsEditorExtensionGalleryUpdateExtensionManually].
+
+You can file issues and contribute to the extension on the [vscode-edge-devtools GitHub repo][GithubMicrosoftVscodeEdgeDevtools].
+
+
+<!-- 4 -->
+## New Fluent UI icons for DevTools
+
+<!-- Title: New look for buttons and menus in Microsoft Edge DevTools -->
+<!-- Subtitle: DevTools has adopted Fluent UI, giving it a more modern look that better aligns with the rest of the Microsoft Edge browser. -->
+
+Microsoft Edge DevTools has adopted [Fluent UI](https://developer.microsoft.com/en-us/fluentui#/), giving buttons and menus a more modern look that better aligns with the rest of the Microsoft Edge browser.
+
+:::image type="complex" source="../../media/2021/07/fluent-ui.msft.png" alt-text="DevTools implemented with Fluent UI design" lightbox="../../media/2021/07/fluent-ui.msft.png":::
+   DevTools implemented with Fluent UI design
+:::image-end:::
+
+<!-- no crbug -->
+
+
+<!-- 5 -->
+## Change the DevTools display language directly from Settings
+
+<!-- Title: DevTools Settings now includes display language -->
+<!-- Subtitle: You can now skip the browser settings and change the DevTools display language directly within DevTools Settings. -->
+
+Previously, to change the display language in DevTools, you had to change the browser language.  Now you can easily switch the locale in DevTools **Settings**, without having to change your browser settings.  To do this, open **Settings**, and then in **Preferences**, select a language from the **Language** drop-down list.
+
+:::image type="complex" source="../../media/2021/07/settings-browser-ui-language.msft.png" alt-text="Changing the DevTools display language directly from DevTools **Settings**" lightbox="../../media/2021/07/settings-browser-ui-language.msft.png":::
+   Changing the DevTools display language directly from DevTools **Settings**
+:::image-end:::
+
+For options for changing settings, navigate to [Customize Microsoft Edge DevTools](../../../customize/index.md).  To review the history of this feature in the Chromium open-source project, navigate to Issue [2882756][CR2882756].
```

Comment:
```suggestion
By default, the DevTools will match your browser language. For more customization options in DevTools, navigate to [Customize Microsoft Edge DevTools](../../../customize/index.md).  To review the history of this feature in the Chromium open-source project, navigate to Issue [2882756][CR2882756].
```

Reply:
@erdraud, please put up a PR that updates https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/customize/localization as well so that we can link there from this announcement instead of the more general Customize overview

Reply:
I'm not committing the above Suggestion; reworded locally.